docs/writing-tests/testharness-api.md

```diff
@@ -248,6 +248,21 @@ promise_test(function(t) {
 }, "Another example");
 ```
 
+## Asynchronous Promise Tests ##
+
+`promise_test` are running sequentially. This might causes timeout when too many
+have to run. `async_promise_test` is a variant, running in parallel.
+
+|              | in sequence | in parallel       |
+| -------------| ----------: | ----------------: |
+| **Function** | test        | async_test        |
+| **Promise**  | promise_test| async_promise_test|
+
+It uses the same API as `promise_test`:
```

Comment:
Perhaps, "`async_promise_test`'s signature is the same as `promise_test`"? Saying that the API is the same is a little confusing.